HelpSource/Classes/QuartzComposerView.schelp

```diff
@@ -66,14 +66,33 @@ Open the currently loaded composition in Quartz Composer. You will need to reloa
 EXAMPLES::
 
 code::
-////////// Simple example
+
+////////// Simple example A (fullscreen)
+(
+var width= 1500, height= 1250; // use a resolution higher then the one of your screen
```

Comment:
This gives us our screen bounds. Please use something like that.

(
var screenBounds, width, height;
screenBounds = Window.screenBounds;
width = screenBounds.width;
height = screenBounds.height;
[screenBounds, width, height].postln;
)